nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc

```diff
@@ -14,8 +14,22 @@ These release notes track the development of the {run-once-operator} for {produc
 
 For an overview of the {run-once-operator}, see xref:../../../nodes/pods/run_once_duration_override/index.adoc#run-once-about_run-once-duration-override-about[About the {run-once-operator}].
 
+[id="run-once-duration-override-operator-release-notes-1-0-1"]
+== {run-once-operator} 1.0.1
+
+Issued: 2023-10-26
+
+The following advisory is available for the {run-once-operator} 1.0.1:
```

Comment:
```suggestion
The following security update advisory is available for the {run-once-operator} 1.0.1:
```
Wondering it should specify security update because it's an RHSA.

Reply:
You're probably right.

Reply:
Andrea says that isn't the standard practice and since it has a link to the errata and a bug fix section I think it's okay they way it is.